index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+     <div class="slider">
+            <div class="slider-block">
+                <img src="img/1.jpg" alt="">
+                <img src="img/2.jpg" alt="">
+                <img src="img/3.jpg" alt="">
+            </div>
+        </div>
+        <div class="button-block">
+            <img class="arrow-prev" src="img/arrow.png" alt="">
```

Comment:
Це кнопка, чи це зображення?
Я б сказав, що таки кнопка.
Може має сенс зробити:
```suggestion
            <button class="arrow-prev" type="button">
```